test/split.jl

```diff
@@ -17,9 +17,14 @@ end
 
 facts("split date operations") do
 
-    context("from and to correctly subset") do
+    context("from and to correctly subset") do # Including zero-length time series
```

Comment:
instead of commenting on the test, it would be better to add that into the `context` string so it becomes transparent. I can change this though, thanks for the tests!